bundles/org.openhab.binding.airq/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,695 @@
+			<channel id="powerFreqSuppression" typeId="Rejection"/>
+			<!-- <channel id="air-Q-Software-Version" typeId="air-Q-Software-Version"/> -->
+			<!-- <channel id="sensors" typeId="sensors"/> -->
+			<channel id="autoDriftCompensation" typeId="AutoDriftCompensation"/>
+			<channel id="autoUpdate" typeId="AutoUpdate"/>
+			<channel id="advancedDataProcessing" typeId="AdvancedDataProcessing"/>
+			<!-- <channel id="Industry" typeId="Industry"/> -->
+			<channel id="ppm_and_ppb" typeId="ppm_and_ppb"/>
+			<channel id="gasAlarm" typeId="GasAlarm"/>
+			<!-- <channel id="id" typeId="id"/> -->
+			<channel id="soundPressure" typeId="SoundInfo"/>
+			<channel id="alarmForwarding" typeId="AlarmForwarding"/>
+			<channel id="userCalib" typeId="usercalib"/>
+			<channel id="initialCalFinished" typeId="InitialCalFinished"/>
+			<channel id="averaging" typeId="Averaging"/>
+			<!-- <channel id="SensorInfo" typeId="SensorInfo"/> -->
+			<channel id="errorBars" typeId="ErrorBars"/>
+
+			<!-- command data, not implemented yet
+				<channel id="getHistoryFiles" typeId="getHistoryFiles"/> -->
+		</channels>
+
+		<properties>
+			<property name="id">Unknown Device ID</property>
+			<property name="hardwareVersion">Unknown Hardware version</property>
+			<property name="softwareVersion">Unknown Software version</property>
+			<property name="sensorList">Unknown sensor list</property>
+			<property name="sensorInfo">No info about sensors</property>
+			<property name="industry">No industry info</property>
+		</properties>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>network-address</context>
+				<label>Network Address</label>
+				<description>The IP Network Address where the air-Q can be reached.</description>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<context>password</context>
+				<label>Password of air-Q</label>
+				<description>The Password of the air-Q device.</description>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+
+	<!-- We have two places where the Device ID is delivered: with the measurement data and with the configuration.
+		We take the info from the configuration and show it as a property, so we don't need this at this moment. We
+		leave this as a reminder in case for some reason it will be needed in future, e.g. when an air-Q device
+		also sends data from other devices (then with another Device ID)
+		<channel-type id="devid" advanced="true">
+		<item-type>String</item-type>
+		<label>Device ID (from Data)</label>
+		<state readOnly="true"></state>
+		</channel-type>
+	-->
+	<channel-type id="status" advanced="false">
+		<item-type>String</item-type>
+		<label>Status of Sensors</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="typps" advanced="false">
+		<item-type>Number:Length</item-type>
+		<label>Average size of Fine Dust</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions